### PR TITLE
Flake tracker error handling

### PR DESCRIPTION
**Pull request by alecps**
### Description

Adds error handling to flake tracker so that CI doesn't fail if there's an issue

### Other changes

None

### Tested

In CI 

### Related issues

- Fixes CI blocker

### Backwards compatibility

Yes

### Documentation

None

### CODE REVIEW

File: packages/flake-tracker/src/manager.js [hunk 1]
```diff
@@ -11,34 +11,54 @@ class FlakeManager {
...
+              return []
+            })
+          )
+        }
       }
+      return new FlakeManager(github)
+    } catch (error) {
+      this.setup = false
+      console.log('Flake tracker setup failed')
+      console.log(error)
```
Comment (gastonponti) [inline packages/flake-tracker/src/manager.js:38]: Maybe I'm missing something, but this build is assigned to a global var, and then something calls `finish()`
In this case, you are not returning anything, so probably that finish will fail.
If I'm not misunderstanding this, you could do 2 things. Re raise the error here. Or, send a new FlakeManager, that only has that flag set (you can add a default flag to the constructor to force skipping the github thing, and just set the flag as false.

Comment (alecps) [reply packages/flake-tracker/src/manager.js:37]: That's why we set this.setup = false  here so that finish() can return immediately when that happens

Comment (alecps) [reply packages/flake-tracker/src/manager.js:38]: Great catch!

**Review by gastonponti — APPROVED** [review]
LGTM!
Thanks for the quick response to this issue!

**Pull request merged**